java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -181,4 +196,48 @@ public CompletableFuture<String> set(
         String[] arguments = ArrayUtils.addAll(new String[] {key, value}, options.toArgs());
         return commandManager.submitNewCommand(SetString, arguments, this::handleStringOrNullResponse);
     }
+
+    @Override
+    public CompletableFuture<Long> zadd(
+            @NonNull String key,
+            @NonNull Map<String, Double> membersScoresMap,
+            @NonNull ZaddOptions options,
+            boolean changed) {
+        String[] changedArg = changed ? new String[] {"CH"} : new String[] {};
+
+        String[] membersScores =
+                membersScoresMap.entrySet().stream()
+                        .flatMap(e -> Stream.of(e.getValue().toString(), e.getKey()))
+                        .toArray(String[]::new);
+
+        String[] arguments =
```

Comment:
Would it be better to use ArrayUtils.addAll to do this? 
```
String[] arguments = ArrayUtils.addAll(new String[] {key}, options.toArgs(), changedArg, membersScores);
```

Reply:
I tried this but I get an error:
```
error: incompatible types: inference variable T has incompatible bounds
        String[] arguments = ArrayUtils.addAll(new String[] {key}, options.toArgs(), changedArg, membersScores);
                                              ^
    lower bounds: String,Object
    lower bounds: String[],String
  where T is a type-variable:
    T extends Object declared in method <T>addAll(T[],T...)
 ```